dbbackup/management/commands/_base.py

```diff
@@ -0,0 +1,18 @@
+from optparse import make_option
+from django.core.management.base import BaseCommand, LabelCommand
+
+
+class BaseDbBackupCommand(LabelCommand):
```

Comment:
I wanted to inherit from `BaseCommand`, but I didn't want to break all. Do you want me to change it?

I purpose to change after all functional tests?

> Le 4 août 2015 à 16:25, benjaoming <notifications@github.com> a écrit :
> 
> In dbbackup/management/commands/_base.py:
> 
> > @@ -0,0 +1,18 @@
> > +from optparse import make_option
> > +from django.core.management.base import BaseCommand, LabelCommand
> > +
> > +
> > +class BaseDbBackupCommand(LabelCommand):
> You inherit from LabelCommand, right? That command class behaves in a certain way, namely that it expects you to pass a label of something.
> 
> The commands in django-dbbackup do not use labels. If you wanted to add that, it would break the API afaik...?
> 
> —
> Reply to this email directly or view it on GitHub.
>